plugins/formal-verification/testData/diagnostics/no_contracts/function_call.fir.diag.txt

```diff
@@ -22,10 +22,10 @@ method global$pkg_$f(local$x: Int) returns (ret$: Int)
 
 method global$pkg_$function_call_nested() returns (ret$: dom$Unit)
 {
-  var anonymous$1: Int
-  var anonymous$2: Int
   var anonymous$3: Int
-  anonymous$1 := global$pkg_$f(0)
-  anonymous$2 := global$pkg_$f(anonymous$1)
-  anonymous$3 := global$pkg_$f(anonymous$2)
+  var anonymous$2: Int
+  var anonymous$1: Int
```

Comment:
Is this reversal intentional?

Reply:
It's a result of the changes in `StmtConversionVisitor.visitFunctionCall` that convert the function arguments into the correct type.

Reply:
Ah... We should probably come up with a unifying strategy there, but this is good for now.